### PR TITLE
atomic registry systemd update

### PR DESCRIPTION
**Pull request by aweiteka**

### CODE REVIEW

File: source/blog/2016-06-17-atomic-registry-systemd.html.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- Ease of configuration that matches a traditional sysadmin experience.
+- Native system logging to journald.
+
+With this approach we leverage docker packaging to deliver applications that *just work*.
+
+Check out the new [systemd deployment](https://github.com/openshift/origin/tree/master/examples/atomic-registry/systemd) for Atomic Registry. The unit files and setup script have been [packaged as an install container](https://hub.docker.com/r/projectatomic/atomic-registry-install/) so you can quickly pull it down and try it out.
+
+## Open Source. Open Process.
+
+As a Red Hat-sponsored project we see the benefits in practicing **open source** as well as **open process**. We committed to this and worked hard to share our public [Trello organization](https://trello.com/atomicopenshift). The [Atomic Registry roadmap card](https://trello.com/c/0hsX6B4G) aggregates all of the planned and in-progress development work. I've been excited about progress on these specific items:
```
Comment (jberkus) [inline source/blog/2016-06-17-atomic-registry-systemd.html.md]: So this really feels like there should be more text here.  WHY are you excited about those specific items?  Can you fill in a little description for people who don't click through to trello?

**Comment by jberkus** [comment]
See line comments.

**Comment by aweiteka** [comment]
@jberkus updated

**Comment by jberkus** [comment]
Looks good.  @bproffitt, please push Monday.

**Pull request merged**